util/logtools/split_log.js

```diff
@@ -113,10 +56,15 @@ const generateFileName = (fightOrZone) => {
   else
     seal = '';
 
-  const idToZoneName = {};
-  for (const zoneName in ZoneId)
-    idToZoneName[ZoneId[zoneName]] = zoneName;
-  const zoneName = idToZoneName[zoneId];
+  let zoneName;
+  if (fightOrZone.zoneName || fightOrZone.name) {
+    zoneName = fightOrZone.zoneName || fightOrZone.name;
```

Comment:
```suggestion
    zoneName = fightOrZone.zoneName ?? fightOrZone.name;
```

Also, is this an unrelated change that just improves names when there's not a zone name? I'm not sure I understand why the logic change is here.  Just trying to clarify my understanding.

Reply:
This is related to what I noted about the discrepancy between `ZoneEncInfo` and `FightEncInfo`. The property `zoneName` is present only on `FightEncInfo`, which is probably a mistake that we should fix separately. (That is, we likely should be explicit about *what each name is*, rather than have *anything* just called a generic "name".)

That being said, you're completely correct that nullish coalescing here is better. I'm not sure why I didn't go with that originally.